PROTOCOL.md

```diff
@@ -0,0 +1,519 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Delta Transaction Log Protocol
+
+- [Overview](#overview)
+- [Delta Table Specification](#delta-table-specification)
+  - [File Types](#file-types)
+    - [Versions](#versions)
+    - [Checkpoints](#checkpoints)
+    - [Data Files](#data-files)
+    - [Last Checkpoint File](#last-checkpoint-file)
+  - [Actions](#actions)
+    - [Change Metadata](#change-metadata)
+      - [Format Specification](#format-specification)
+    - [Add File and Remove File](#add-file-and-remove-file)
+    - [Transaction Identifiers](#transaction-identifiers)
+    - [Protocol Evolution](#protocol-evolution)
+    - [Commit Provenance Information](#commit-provenance-information)
+- [Appendix](#appendix)
+  - [Per-file Statistics](#per-file-statistics)
+  - [Schema Serialization Format](#schema-serialization-format)
+    - [Primitive Types](#primitive-types)
+    - [Struct Type](#struct-type)
+    - [Struct Field](#struct-field)
+    - [Array Type](#array-type)
+    - [Map Type](#map-type)
+    - [Example](#example)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Overview
+This document is a specification for the Delta Transaction Protocol, which brings [ACID](https://en.wikipedia.org/wiki/ACID) properties to large collections of data, stored as files, in a distributed file system or object store. The protocol was designed with the following goals in mind:
+
+- **Serializable ACID Writes** - multiple writers can concurrently modify a Delta table while maintaining ACID semantics.
+- **Snapshot Isolation for Reads** - readers can read a consistent snapshot of a Delta table, even in the face of concurrent writes.
+- **Scalability to billions of partitions or file** - queries against a Delta table can be planned on a single machine or in parallel.
+- **Self describing** - all metadata for a Delta table is stored alongside the data, static tables can be copied or moved using standard tools.
```

Comment:
Missing a "so" or something after the comma? You could also make it clearer that static tables can be copied or moved by just copying/moving a directory.

Reply:
Rephrased.